src/libsync/networkjobs.cpp

```diff
@@ -589,6 +590,42 @@ bool PropfindJob::finished()
 
 /*********************************************************************************************/
 
+AvatarJob::AvatarJob(AccountPtr account, QObject *parent)
+    : AbstractNetworkJob(account, QString(), parent)
+{
+    _avatarUrl = Utility::concatUrlPath(account->url(), QString("remote.php/dav/avatars/%1/128.png").arg(account->davUser()));
+}
+
+void AvatarJob::start()
+{
+    QNetworkRequest req;
+    setReply(davRequest("GET", _avatarUrl, req));
+    setupConnections(reply());
+    AbstractNetworkJob::start();
+}
+
+bool AvatarJob::finished()
+{
+    int http_result_code = reply()->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+
+    QPixmap avPixmap;
```

Comment:
This seems to cause the FolderManTest to fail on your PR, since that needs a QGuiApplication. And you don't seem to need a QPixmap either, since you're creating a new one in `circleMask` anyway, so storing it as a QImage could be enough to fix that test.